stock_picking_progress/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+{
+    "name": "Stock Picking Progress",
+    "summary": "Compute the stock.picking progression",
+    "version": "16.0.1.0.0",
+    "category": "Inventory",
+    "website": "https://github.com/OCA/stock-logistics-workflow",
+    "author": "mmequignon, Camptocamp SA, Odoo Community Association (OCA)",
```

Comment:
@mmequignon Was this module existing prior to Camptocamp? Otherwise should be:
```suggestion
    "author": "Camptocamp SA, Odoo Community Association (OCA)",
```

Reply:
Anyway, to be consistent with other modules, it's better to have `Camptocamp` instead of `Camtocamp SA`